Core JS/index.js

```diff
@@ -0,0 +1,12 @@
+var totalFormSubmitted = 0;
```

Comment:
We can leverage submittedData -> array.count to get the totalFormSubmitted value